src/main/java/technology/tabula/extractors/OcrConverter.java

```diff
@@ -0,0 +1,58 @@
+package technology.tabula.extractors;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.imageio.spi.IIORegistry;
+import javax.imageio.spi.ImageWriterSpi;
+
+import com.github.jaiimageio.impl.plugins.tiff.TIFFImageWriterSpi;
+
+// Tess4j imports
+import net.sourceforge.tess4j.ITesseract;
+import net.sourceforge.tess4j.Tesseract;
+import net.sourceforge.tess4j.TesseractException;
+import net.sourceforge.tess4j.util.LoadLibs;
+import net.sourceforge.tess4j.util.PdfUtilities;
+
+public class OcrConverter {

```

Comment:
It would be great to expose OCR functionality in the [command-line app](https://github.com/tabulapdf/tabula-java/blob/master/src/main/java/technology/tabula/CommandLineApp.java)